src/components/GraphTutorialPopover/GraphTutorialPopover.css

```diff
@@ -6,9 +6,10 @@
 
 .helpIcon {
   color: grey !important;
-
+  margin-top: -10px;
+  text-align: center;
 }
 
 .helpToolTip {
-  padding: 20px;
+  padding: 0px;
```

Comment:
instead of 0px padding you might as well remove this class altogether.

Reply:
The 0px overrides the inheritance from it's parent division. Sadly, we cant remove it.